vocabularies/voc4cat/0000016.ttl

```diff
@@ -1,10 +1,13 @@
 PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://w3id.org/nfdi4cat/voc4cat_0000016>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0000016"^^xsd:token ;
+    rdfs:isDefinedBy <https://www.w3.org/TR/2010/NOTE-curie-20101216/> ;
+    skos:broadMatch <https://www.w3.org/ns/sosa/Sample> ;
```

Comment:
This should not be here. Maybe there is a bug in the code for mapping external vocuabularies. I will check and eventually remove the external mapping and add it in another PR later.